<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CamShakeClass</class>
 <widget class="QMainWindow" name="CamShakeClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>549</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>549</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CamShake</string>
  </property>
  <property name="windowIcon">
   <iconset resource="camshake.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.950000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 9pt &quot;Consolas&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>480</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;Consolas&quot;;</string>
       </property>
       <property name="title">
        <string>Settings</string>
       </property>
       <widget class="MyQPlainTextEdit" name="plainTextEdit" native="true">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>25</y>
          <width>71</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="MyQPlainTextEdit" name="plainTextEdit_2" native="true">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>65</y>
          <width>71</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="MyQPlainTextEdit" name="plainTextEdit_3" native="true">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>105</y>
          <width>71</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="MyQPlainTextEdit" name="plainTextEdit_7" native="true">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>140</y>
          <width>71</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>LEFT</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>RIGHT</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>UP</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>DOWN</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ContinuousBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>300</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>100</horstretch>
          <verstretch>20</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Consolas&quot;;</string>
        </property>
        <property name="text">
         <string>ContiMode</string>
        </property>
       </widget>
       <widget class="QPushButton" name="DiscreteBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>350</y>
          <width>101</width>
          <height>30</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>DiscMode</string>
        </property>
       </widget>
       <widget class="QPushButton" name="StopBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>400</y>
          <width>101</width>
          <height>30</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="Face">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>190</y>
          <width>89</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Face</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="Fist">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>220</y>
          <width>89</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Fist</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_detection">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/CamShake/bk);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyQPlainTextEdit</class>
   <extends>QWidget</extends>
   <header>myqplaintextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="camshake.qrc"/>
 </resources>
 <connections/>
</ui>
